### DISCUSSION TITLE
How to build the docs locally?

### DISCUSSION THREAD

**Discussion by blackjyn** [typst/typst#3457] [General]
I clone the repo and successfully build typst binary with no issue.

However, I'm struggle to build the docs locally that mimics the online docs at typst.app 

Several docs documents are in markdowns, but other docs like Math docs are not there. 

So how do I build the docs locally , mimics the docs at typst.app?

I'm new to Rust btw.

Thanks

**Comment by laurmaedje** [comment]
Some of the docs are in markdown, others are inline with the Rust code. While all of the content is in this repository, it is not currently possible to build the docs locally as they are generated alongside our closed-source website. We plan to improve that in the future.

**Reply by blackjyn** [reply]
Ah.. thanks 
At least it is clear now.
